zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -55,7 +53,7 @@ class Handlers(jsonGenerator: ZipkinJson, mustacheGenerator: ZipkinMustache, que
 
     def apply(response: Response) {
       response.setContentType(typ)
-      response.content = ChannelBuffers.wrappedBuffer(content)
+      response.content = Buf.ByteArray(content)
```

Comment:
Sorry if I'm narrow on the context. I'll look closer and follow up tomorrow, but here's a quick answer for now.

To get a Buf from an InputStream we have to use a Reader.

```
import com.twitter.io.Reader
val reader: Reader = Reader.fromStream(stream)
val buf: Future[Buf] = Reader.readAll(reader)
```

This gives a Future[Buf], which you could flatMap to get at the Buf.